README.md

```diff
@@ -4,12 +4,53 @@
 [![Coverage report](https://gitlab.laas.fr/loco-3d/sl1m/badges/master/coverage.svg?job=doc-coverage)](http://projects.laas.fr/gepetto/doc/loco-3d/sl1m/master/coverage/)
 
 
-Python dependencies:
-	scipy
-	quadprog
-	matplotlib
-	pycddlib
-	cvxpy
+## Python dependencies:
+	
+	python -m pip install scipy
+	sudo apt install robotpkg-py38-quadprog
+	python -m pip install matplotlib
+	python -m pip install pycddlib
+	python -m pip install cvxpy
 
 :warning: this package requires gurobi :warning:
 A free academic license can be obtained at https://www.gurobi.com
+
+## Demo dependencies:
+
+These are the dependencies for the robot and rbprm server.
+```
+sudo apt install -y robotpkg-py38-hpp-rbprm \
+					robotpkg-py38-hpp-rbprm-corba \
```

Comment:
hpp-rbprm-corba depends on anymal-rbprm, solo-rbprm, talos-rbprm and solo-rbprm, so you only need to install this one

Reply:
```
$ sudo apt install robotpkg-py38-hpp-rbprm-corba
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  robotpkg-py38-anymal-rbprm+doc robotpkg-py38-hpp-rbprm robotpkg-py38-talos-rbprm
The following NEW packages will be installed:
  robotpkg-py38-anymal-rbprm+doc robotpkg-py38-hpp-rbprm robotpkg-py38-hpp-rbprm-corba
  robotpkg-py38-talos-rbprm
0 upgraded, 4 newly installed, 0 to remove and 3 not upgraded.
Need to get 58.1 MB of archives.
After this operation, 0 B of additional disk space will be used.
Do you want to continue? [Y/n] 
```
It seems that the dependency to solo is missing This is why I added all this, I will fix the install and add a trouble-shooting installation section